iset.mm

```diff
@@ -72370,6 +72372,21 @@ Similar to the Axiom of Choice (first form) of [Enderton] p. 49.
 =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
 $)
 
+  $c Ap $.
+
+  $( Apartness predicate symbol. $)
+  wap $a wff R Ap A $.
+
+  ${
+    $d x y z A $.  $d x y z R $.
+    $( Apartness predicate.  A relation ` R ` is an apartness if it is
+       irreflexive, symmetric, and cotransitive.  (Contributed by Jim Kingdon,
+       14-Feb-2025.) $)
+    df-pap $a |- ( R Ap A <-> ( ( R C_ ( A X. A ) /\ A. x e. A -. x R x ) /\
+      ( A. x e. A A. y e. A ( x R y -> y R x ) /\
+      A. x e. A A. y e. A A. z e. A ( x R y -> ( x R z \/ y R z ) ) ) ) ) $.
+  $}
```

Comment:
Are you sure about the "cotransitive" part?
Shouldn't it be `( x R z -> ( x R z \/ y R z ) )` instead ? ( `x R z` instead of `x R y`)

Reply:
You're right, sorry.

There might be a nice definition without dummy variables, just like [`df-er`](https://us.metamath.org/mpeuni/df-er.html).
Maybe something like
```
( R Ap A <-> ( ( R C_ ( ( A X. A ) \ _I ) /\ `' R C_ R /\ ( ( ( ( A X. A ) \ R ) o. ( ( A X. A ) \ R ) ) u. R ) = ( A X. A ) )
```
If I'm not mistaken again

Reply:
Heh, I'll let others judge about "nice" but since we can have multiple definitions (one as the `$a`, the others as theorems proved from it) we could always add others.

Just to glance at the proposal above, I think maybe it needs excluded middle to work but I could easily be mistaken on that. In any event, seems like something for a future pull request.